src/axom/core/tests/core_array.hpp

```diff
@@ -1931,4 +1931,18 @@ TEST(core_array, checkVariadicCtors)
   Array<int, 3> arr12(s, s, s);
 }
 
+//------------------------------------------------------------------------------
+
+TEST(core_array, check_subspan_range)
+{
+  int m = 10;
+  int n = 3;
+  Array<int> arr(m);
+  ArrayView<int> arrv1(arr);
+  ArrayView<int> arrv2 = arrv1.subspan(n);
+  EXPECT_EQ(arrv2.size() + n, arrv1.size());
+  EXPECT_GE(&arrv2[0], &arr[0]);
+  EXPECT_LE(&arrv2[arrv2.size() - 1], &arr[arr.size() - 1]);
```

Comment:
Similarly, check the address of the last element of arrv2 here.

Reply:
Aggeed.